fast/stages/2-networking-c-nva/net-landing.tf

```diff
@@ -129,3 +129,64 @@ module "landing-firewall" {
     rules_folder  = "${var.factories_config.data_dir}/firewall-rules/landing"
   }
 }
+
+# Management VPC
+
+
+module "management-vpc" {
+  source     = "../../../modules/net-vpc"
+  project_id = module.landing-project.project_id
+  name       = "prod-management-0"
+  mtu        = 1500
+  dns_policy = {
+    inbound = true
```

Comment:
Inbound policy are unlikely to be required in this VPC

Reply:
fixed on https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/2180/commits/76e271e9c89f3acd28e37cf1e1d2b5e3f95153c6